data-exporter/src/main/java/org/reactome/release/dataexport/EuropePMC.java

```diff
@@ -124,10 +124,8 @@ private String getEuropePMCProfileXML() {
 		nodeNameToValue.put("id", Integer.toString(reactomeProviderID));
 		nodeNameToValue.put("resourceName", "Reactome");
 		nodeNameToValue.put("description",
-			"Reactome is a free, open-source, curated and peer-reviewed pathway database. " +
-			"Our goal is to provide intuitive bioinformatics tools for the visualization, " +
-			"interpretation and analysis of pathway knowledge to support basic research, genome analysis, " +
-			"modeling, systems biology and education.");
+			"Reactome is a free, open-source, curated and peer-reviewed pathway database."
```

Comment:
Did we lose this the mission statement?

Reply:
No, we still have this full mission statement but we decided to go with a shorter version for use in Europe PMC.